### PR TITLE
Updated configuration on where output files should be treated

### PR DESCRIPTION
**Pull request by egede**
Added the various batch backends into configuration. In the longer term some default action might be preferable to define.

### CODE REVIEW

**Comment by alexanderrichards** [comment]
Can one of the admins verify this patch?

**Comment by alexanderrichards** [comment]
alex test

**Comment by alexanderrichards** [comment]
retest this please

**Comment by mesmith75** [comment]
Do we want to add SGE to the MassStorageFile and SharedFile configs as well?

File: python/Ganga/__init__.py [hunk 1]
```diff
@@ -715,7 +756,16 @@ def filefilter(fn):
                         'defaultProtocol': defaultMassStorageProto},
                        massStorageFileExt)
 
-sharedFileBackendPost = {'LSF': 'client', 'LCG': 'client', 'ARC': 'client', 'Dirac': 'client', 'PBS': 'client', 'Interactive': 'client', 'Local': 'client', 'CREAM': 'client'}
+sharedFileBackendPost = {'LSF': 'WN',
+                         'LCG': 'WN',
```
Comment (mesmith75) [inline python/Ganga/__init__.py]: Is this LCG one correct?

**Pull request merged**